memberListActivities/index.tsx

```diff
@@ -164,8 +170,15 @@ export default definePlugin({
                 }
 
                 if (application) {
+                    // XBox application id
+                    if (application_id === "438122941302046720") {
```

Comment:
Personally I think using a ternary for the src variable would be a lot cleaner, then you don't have the duplicate icon.push line and you don't have a duplicate variable declaration. You can also use the Application name instead and remove the comment for readability.
The end result would be something like.
```JS
if (application) {
    const src = application.name === "Xbox" ? "https://discord.com/assets/9a15d086141be29d9fcd.png" : `https://cdn.discordapp.com/app-icons/${application.id}/${application.icon}.png`;
    icons.push(<img src="{src}" alt="{application.name}" />);
}
```
Makes this go from adding 7 lines to only modifying 1.

Reply:
unfortunately we can't use application.name as it is set to the game ur playing, so we have to use id instead, but i've done pretty much what u suggested